### PR TITLE
feat(uapi): add mfa procedure

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: console/my-account/how-to/accept-invitation-new-scw-account.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  paragraph: This page explains how to change the color scheme of the Scaleway console
+tags: color-scheme change color theme dark-mode change-color
+dates:
+  validation: 2023-06-05
+  posted: 2023-06-15
+categories:
+  - console
+---
+
+When a Scaleway user invites you to join their Organization you receive an invitation via email. If do not yet have a Scaleway account upon receiving this email, you will not be able to effectively join the Organization.
```
Comment (crlptl) [inline console/my-account/how-to/accept-invitation-new-scw-account.mdx:15]: Missing the you: "If YOU do not have..."

File: console/my-account/how-to/accept-invitation-new-scw-account.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
```
Comment (crlptl) [inline console/my-account/how-to/accept-invitation-new-scw-account.mdx:16]: Something I don't understand in your doc: if we are talking about an organization that does not enforce MFA, the sentence "If do not yet have a Scaleway account upon receiving this email, you will not be able to effectively join the Organization." is false.

File: console/my-account/how-to/accept-invitation-new-scw-account.mdx (hunk 1 above)
Comment (ldecarvalho-doc) [reply console/my-account/how-to/accept-invitation-new-scw-account.mdx:15]:
```suggestion
When a Scaleway user invites you to join their Organization you receive an invitation via email. If you do not yet have a Scaleway account upon receiving this email, you will not be able to effectively join the Organization.
```

File: console/my-account/how-to/accept-invitation-new-scw-account.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: How to activate a new account after being invited to an Organization
+  description: This page explains how to activate a new account after being invited to an Organization
+  h1: How to activate a new account after being invited to an Organization
+  paragraph: This page explains how to change the color scheme of the Scaleway console
+tags: color-scheme change color theme dark-mode change-color
```
Comment (RoRoJ) [inline console/my-account/how-to/accept-invitation-new-scw-account.mdx:7]: think the tags need updating

File: console/my-account/how-to/accept-invitation-new-scw-account.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Click **Accept Invitation** in the email you received inviting you to join an existing Scaleway Organization.
+    <Message type="important">
+      When you click the link, you are redirected to the Scaleway console. Since you do not have a Scaleway account at this point, you will not be able to log into the console.
+    </Message>
+2. Go back to your personal email mailbox.
+3. Check if you have received an email from Scaleway with the following subject: **[Action Required] Confirm Your Scaleway Account**.
+4. Open the email if you have received it.
+    <Message type="important">
+      If you have not received the email, check that it has not been sent to SPAM. If it is not in your SPAM folder, wait a while longer to see if you will receive the email. You can also contact the person who invited you to re-send the invitation. If none of these options work, open a ticket and [contact our support team](https://console.scaleway.com/support/tickets).
```
Comment (RoRoJ) [inline console/my-account/how-to/accept-invitation-new-scw-account.mdx:27]: Will they be able to open a ticket if they don't have an account?

File: console/my-account/how-to/accept-invitation-new-scw-account.mdx [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    </Message>
+5. Read through the Scaleway Terms of Service and Data Protection Agreement. You can find them in the PDFs attached to the email.
+6. Click **I Accept Scaleway's Terms Of Service And I Accept Scaleway's Data Protection Agreement**. You are redirected to a page in the Scaleway console, confirming you have successfully accepted the Terms of Service and Data Protection Agreement.
+7. Click **Go to Scaleway console**. You are redirected to the log in screen.
```
Comment (RoRoJ) [inline console/my-account/how-to/accept-invitation-new-scw-account.mdx:31]:
```suggestion
7. Click **Go to Scaleway console**. You are redirected to the login screen.
```

**Review by ofranc — APPROVED** [review]

**Review by ofranc — APPROVED** [review]

File: console/my-account/how-to/accept-invitation-new-scw-account.mdx (hunk 5 above)
Comment (ldecarvalho-doc) [reply console/my-account/how-to/accept-invitation-new-scw-account.mdx:31]: yes, I checked!

File: console/my-account/how-to/accept-invitation-new-scw-account.mdx (hunk 4 above)
Comment (ldecarvalho-doc) [reply console/my-account/how-to/accept-invitation-new-scw-account.mdx:27]: yes, I checked!

**Review by RoRoJ — APPROVED** [review]

**Pull request closed without merging**